pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -142,6 +158,47 @@ type TargetCluster struct {
 	Replicas int32 `json:"replicas,omitempty"`
 }
 
+// GracefulEvictionTask represents an graceful eviction task.
+type GracefulEvictionTask struct {
+	// Name of the cluster where the eviction perform from.
+	Name string `json:"name"`
```

Comment:
Would it be better to use `ClusterName`? Because there is no cluster in the current context.

Reply:
Good point. This field should represent which cluster the referencing resource is evicted from.

Reply:
Done. `Name` --> `FromCluster`.